arctic/store/version_store.py

```diff
@@ -293,7 +313,7 @@ def _write_handler(self, version, symbol, data, **kwargs):
             handler = self._bson_handler
         return handler
 
-    def read(self, symbol, as_of=None, date_range=None, from_version=None, allow_secondary=None, **kwargs):
+    def read(self, symbol, as_of=None, date_range=None, from_version=None, allow_secondary=None, all_metadata=False, **kwargs):
```

Comment:
naming of the parameter, all_metadata:
consider:
 include_meta_history_in_time
 metadata_history
 metadata_in_time
etc.